nemo/core/config/hydra_runner.py

```diff
@@ -91,18 +91,18 @@ def wrapper(cfg_passthrough: Optional[DictConfig] = None) -> Any:
 
                 # Wrap a callable object with name `parse_args`
                 # This is to mimic the ArgParser.parse_args() API.
-                class _argparse_wrapper:
-                    def __init__(self, arg_parser):
-                        self.arg_parser = arg_parser
-                        self._actions = arg_parser._actions
+                def parse_args(self, args=None, namespace=None):
+                    return parsed_args
 
-                    def parse_args(self, args=None, namespace=None):
-                        return parsed_args
+                parsed_args.parse_args = parse_args
 
                 # no return value from run_hydra() as it may sometime actually run the task_function
                 # multiple times (--multirun)
+                argparse_wrapper = parsed_args
```

Comment:
Why do you need this, can't you just pass in `parsed_args`?

Reply:
Sure, I've kept it there cause we needed to wrap it in the past. This is for me to remember to add back the wrapper in the future.